src/components/Identity/createMigration.js

```diff
@@ -10,7 +10,10 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { cookieJar } from "../../utils";
+import { getApexDomain, cookieJar } from "../../utils";
+// TODO: We are already retrieving the apex in core; find a way to reuse it.
+// Maybe default the domain in the cookieJar to apex while allowing overrides.
+const apexDomain = getApexDomain(window, cookieJar);
```

Comment:
We have the current apex domain already calculated up in core (https://github.com/adobe/alloy/blob/master/src/core/index.js#L53) if you want to inject it down.

Reply:
> We have the current apex domain already calculated up in core (https://github.com/adobe/alloy/blob/master/src/core/index.js#L53) if you want to inject it down.

@Aaronius, what about the comment I added? Do you think defaulting it in the `cookieJar` is a valid approach?

Reply:
Yeah, that's an interesting concept. I think it's a good idea.

Reply:
I will go ahead and merge this PR. Created a ticket to address the APEX default: https://jira.corp.adobe.com/browse/CORE-39130